src/now.rs

```diff
@@ -22,6 +22,17 @@ pub(crate) use self::sys::*;
 )))]
 mod sys {
     pub(crate) fn system_time() -> std::time::SystemTime {
+        // `SystemTime::now()` should continue to panic on this exact target in the future
+        // as well; Instead of letting `std` panic, we panic first with a more informative
+        // error message.
```

Comment:
```suggestion
        // `SystemTime::now()` should continue to panic on this exact target in
        // the future as well; Instead of letting `std` panic, we panic first
        // with a more informative error message.
```

(line length)